selene-lib/src/lints/global_usage.rs

```
@@ -36,7 +28,9 @@ impl Lint for GlobalLint {
 
     fn new(config: Self::Config) -> Result<Self, Self::Error> {
         Ok(GlobalLint {
-            ignore_pattern: Regex::new(&config.ignore_pattern)?,
+            ignore_pattern: config
+                .ignore_pattern
+                .and_then(|ignore_pattern| Regex::new(&ignore_pattern).ok()),

```

We're ignoring the error!